x-pack/plugins/reporting/server/export_types/common/pdf/integration_tests/pdfmaker.test.ts

```diff
@@ -5,16 +5,15 @@
  * 2.0.
  */
 
-import { createMockLayout } from '../../../../../screenshotting/server/layouts/mock';
-import { PdfMaker } from './';
+import { createMockLayout } from '../../../../../../screenshotting/server/layouts/mock';
+import { PdfMaker } from '../';
```

Comment:
This test integrates with an NPM module, which does most of the work. For this reason, it should be an integration test. It previously was flaky because it was written as a plain unit test that has a 5-second timeout.